mcpscanner/docker/entrypoint.py

```
@@ -0,0 +1,232 @@
+    spec = f"{package}=={version}" if version else package
+    logger.info("Downloading %s", spec)
+
+    # Try source distribution first
+    cmd_sdist = [
+        sys.executable, "-m", "pip", "download",
+        "--no-deps",
+        "--no-binary", ":all:",
+        "--dest", DOWNLOAD_DIR,
+        spec,
+    ]
+    result = subprocess.run(cmd_sdist, capture_output=True, text=True)
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)